core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,13 @@ var AdminPage = function() {
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="$ctrl.statusMessage"]'));
 
+  // Viewing roles can be done by two methods: 1. By roles 2. By username
```

Comment:
I don't understand what it means by "Viewing roles can be done by roles"

Reply:
Roles can be viewed by roles or username...hope this screenshot helps
![Screenshot 2019-10-22 at 1 06 56 AM](https://user-images.githubusercontent.com/30312043/67237637-45b8e780-f469-11e9-817a-b9324a465449.png)

Reply:
Ah I see, I don't see how we can write this better. I think let's leave it as is.